src/MarketingPageAcceptanceTests.Actions/Pages/PreviewPage.cs

```diff
@@ -163,5 +163,10 @@ public bool IsRequiresHscnDisplayed(String section)
                 .FindElements(pages.PreviewPage.RequiresHscn)
                 .Count > 0;
         }
+
+        public bool RoadmapSectionDisplayed()
```

Comment:
is it worth making this generic, and passing "roadmap" in?

Reply:
I'll modify it for a future story